Makefile

```diff
@@ -82,6 +82,14 @@ ifeq ($(DISABLE_CGO), 1)
 	override BUILDTAGS = exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
 endif
 
+ARCHES=amd64 s390x ppc64le
+MA_IMAGE := $(REGISTRY)/$(REPO):$(TAG)
```

Comment:
Maybe the caller could just set `MA_IMAGE` and `EXTRA_MA_IMAGE` (or whatever they end up being named); I don’t think any of the `REGISTRY`/`REPO`/`TAG` individual variables make anything simpler, it’s just more variables to track.

Reply:
yeah, probably you are right :) Will think how to reduce env variable number and make them self-understandable